include/diy/master.hpp

```diff
@@ -52,7 +52,7 @@ namespace diy
       using Skip = std::function<bool(int, const Master&)>;
 
       struct SkipNoIncoming;
-      struct NeverSkip { bool    operator()(int i, const Master& master) const   { return false; } };
+      struct NeverSkip { bool    operator()(int i, const Master& master) const   { (void)i; (void)master; return false; } };
```

Comment:
Let's be consistent and simply not name these kinds of unused arguments in the function's parameter list.